pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -153,7 +165,11 @@ public PendingAckHandleImpl(PersistentSubscription persistentSubscription) {
 
         this.pendingAckStoreProvider = this.persistentSubscription.getTopic()
                         .getBrokerService().getPulsar().getTransactionPendingAckStoreProvider();
+        transactionOpTimer = persistentSubscription.getTopic().getBrokerService().getPulsar().getTransactionTimer();
+        init();
+    }
 
+    private void init() {
```

Comment:
Do we need to make the method `sync`?

Reply:
No, this method does not need to be synced. The method `init ` is called only when the PendingAckHandle is created or the last one failed.